packages/phone_log/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [0.0.3] - August 15th, 2018.
```

Comment:
You also need to change the pubspec.yaml file to rev the version to 0.0.3

Reply:
Done. Thank you for catching this!